lib/pure/collections/sequtils.nim

```diff
@@ -43,8 +43,8 @@ proc concat*[T](seqs: varargs[seq[T]]): seq[T] =
       result[i] = itm
       inc(i)
 
-proc count*[T](list: seq[T], item: T): int =
-  ## Count the occurrences of the item `item` in the sequence `list`.
+proc count*[T](s: openArray[T], x: T): int =
+  ## Count the occurrences of the item `x` in the sequence `s`.
```

Comment:
But it's not a sequence anymore.

Reply:
Container?

Reply:
my point is that the comment is wrong.

Reply:
I understood that :) I'm asking what word should be used in comments to better describe it. Is it "containter" or do you prefer something else?

Reply:
container is fine.

Reply:
A new commit submitted, changes this to "container", and makes some other minor documentation fixes.